catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -386,7 +388,7 @@ protected void postEvent(String topic, Metacard card, Metacard oldCard) {
     Event event = new Event(topic, properties);
     eventAdmin.postEvent(event);
 
-    LOGGER.debug("EXITING: {}", methodName);
+    LOGGER.debug(EXITING, methodName);
```

Comment:
✏️ All of these entering/exiting log messages should be at trace: https://codice.atlassian.net/wiki/spaces/DDF/pages/7045131/Log+Levels.